### ISSUE TITLE
Issues After Update

### ISSUE THREAD

**Issue by locass3** [qtile/qtile#2252]
```
locass03@archlinux ~> qtile -l ERROR
Traceback (most recent call last):
  File "/usr/bin/qtile", line 33, in <module>
    sys.exit(load_entry_point('qtile==0.17.1.dev30+g6248c7a3', 'console_scripts', 'qtile')())
  File "/usr/lib/python3.9/site-packages/libqtile/scripts/main.py", line 59, in main
    options.func(options)
AttributeError: 'Namespace' object has no attribute 'func'
```

I've updated to the latest kernel version `5.11.1-arch1-1` and  `qtile (0.16.1-2 -> 0.17.0-1)`. I am getting the error above. I've tried both qtile from the official release and the latest git request  and both have the same error using the default config. 

Any ideas on how to fix this?

Many thanks

**Comment by tych0** [comment]
You need to do `qtile start` now in order to start qtile, see: https://github.com/qtile/qtile/blob/master/CHANGELOG#L34

Though I would have expected 0.17.0 to have some backwards compatibility.

**Comment by ramnes** [comment]
Yeah but we shouldn't have a traceback anyway

**Comment by tych0** [comment]
I suspect it's because it was not just bare `qtile`, but `qtile -l ..`. We can't really fix this in the released version, short of tagging a 0.17.1 or similar, which maybe we should do...

**Comment by elParaguayo** [comment]
If we tag 0.17.1 on current master then that brings in the `dbus-next` merge. I have no issues with that but I just wanted to point out that some concern was raised (in comments on #1994) about `python-dbus-next` not being in official repos yet.

**Comment by ramnes** [comment]
It's actually a bug that was present in 0.17.0 but that got fixed with #2143.

**Comment by locass3** [comment]
Thank you all for your contributions!
